waspc/examples/todoApp/todoApp.wasp

```diff
@@ -105,6 +105,12 @@ query getTasks {
   entities: [Task]
 }
 
+api fooBar {
+  fn: import { fooBar } from "@server/apis.js",
+  entities: [Task],
+  httpRoute: (GET, "/foo/bar")
```

Comment:
Should we maybe prefix all api routes with `api`? Since operations are under `operations`?

Reply:
I vote against it since users might need ultra specific callback path and then they are doomed 😢

Reply:
Ok, makes sense